17: My Solution requires `mut tx`.

The borrow checker does not seem happy as is.

```sh
error[E0596]: cannot borrow immutable local variable `rx` as mutable
  --> src/main.rs:30:21
   |
27 |     let Serial { tx, rx } = Serial::new().unwrap();
   |                      -- consider changing this to `mut rx`
...
30 |     let mut bytes = rx.bytes();
   |                     ^^ cannot borrow mutably

error[E0596]: cannot borrow immutable local variable `tx` as mutable
  --> src/main.rs:45:13
   |
27 |     let Serial { tx, rx } = Serial::new().unwrap();
   |                  -- consider changing this to `mut tx`
...
45 |             tx.write(byte).wait();
   |             ^^ cannot borrow mutably
error: aborting due to 2 previous errors
error: Could not compile `async`.
To learn more, run the command again with --verbose.
```

--- a/src/17-async-io-the-future/my-solution.md
+++ b/src/17-async-io-the-future/my-solution.md
@@ -3,7 +3,7 @@
 ``` rust
 pub fn main() -> ! {
     let mut timer = Timer::new().unwrap();
-    let Serial { tx, rx } = Serial::new().unwrap();
+    let Serial { mut tx, mut rx } = Serial::new().unwrap();
 
     let mut periodic = timer.periodic(100);
     let mut bytes = rx.bytes();

